### DISCUSSION TITLE
LARAGON - Menu/MySQL/phpMyAdmin links to https://forum.laragon.org/topic/98  ???

### DISCUSSION THREAD

**Discussion by Sheldon-Monk** [leokhoa/laragon#516] [Q&A] [answered]
The Menu/MySQL/phpMyAdmin path links to https://forum.laragon.org/topic/98 (the "I regret to inform you that due to financial difficulties... page), NOT to phpMyAdmin.  Why is that?  More importantly, how do I open phpMyAdmin?

**Comment by LrkDev** [comment, marked as answer]
Make sure you have installed "phpMyAdmin" by going to "Menu/Tools/Quick Add/phpMyAdmin"

You can also unzip phpMyAdmin into this path "laragon\etc\apps\phpMyAdmin"

**Reply by web-v2** [reply]
Luego de agregar phpmyadmin y tratar de acceder me sale este error en el navegador:  
`Fatal error: Unparenthesized `a ? b : c ? d : e` is not supported. Use either `(a ? b : c) ? d : e` or `a ? b : (c ? d : e)` in D:\APPs\laragon\etc\apps\phpMyAdmin\libraries\classes\DatabaseInterface.php on line 615`

**Comment by Sheldon-Monk** [comment]
Just curious:  any reason it wouldn't have been included in laragon install??

**Reply by Afumoons** [reply]
I think it's because using HeidiSQL (which is included) instead of phpMyAdmin..

**Comment by ShivamKushwahCoding** [comment]
Make sure you have installed "phpMyAdmin" by going to "Menu/Tools/Quick Add/phpMyAdmin"
And Also Make sure you have installed "mysql" by going to "Menu/Tools/Quick Add/mysql"

This worked for me.
